go.mod

```diff
@@ -49,6 +51,7 @@ require (
 	github.com/go-sql-driver/mysql v1.9.2
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/snappy v1.0.0
+	github.com/golangci/golangci-lint/v2 v2.1.5
```

Comment:
This shouldn't be here 🤔  This should be a tooling dependency...

Reply:
It is a tooling dependency, but it doesn't work properly unless you either entirely externalize it or run it from the package root with the root's go.mod. This was a part of the back-and-forth with the golangci-lint v2 PRs.